pyiron_base/job/generic.py

```diff
@@ -500,7 +501,20 @@ def copy_to(self, project=None, new_job_name=None, input_only=False, new_databas
         if project is None and new_job_name is None:
             raise ValueError("copy_to requires either a new project or a new_job_name.")
 
-        in_same_project = project is None or project.path == self.project.path
+        if isinstance(project, self.project.__class__):
+            in_same_project = project.path == self.project.path
+            project = project or self.project
+            job_table = project.job_table(recursive=False)
+        elif isinstance(project, self.project_hdf5.__class__):
+            in_same_project = project.path == self.project_hdf5.path
```

Comment:
Is there a guideline for when to use `._hdf5` and when to use its `.project_hdf5` wrapper?

Reply:
The suggestion is always use `project_hdf5`.